nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/listener/test/SimpleListenerTest.java

```diff
@@ -117,8 +119,8 @@ public void testDocumentRemovedCommentEventListener() throws Exception {
         waitForAsyncExec();
 
         // Did all the comments have been deleted?
-        comments = commentManager.getComments(session, doc.getId());
-        assertTrue(comments.isEmpty());
+        DocumentModelList documents = session.query(String.format("SELECT * FROM Comment WHERE %s = '%s'", NXQL.ECM_ANCESTORID, doc.getId()));
+        assertTrue(documents.isEmpty());
```

Comment:
Why this change?
Note: this test works only with new service with this query.

Reply:
yes, i agree this was an oversight on my previous `TreeCmmentManager#getComments` version